src/Text/Pandoc/Writers/Markdown/Inline.hs

```diff
@@ -469,6 +550,9 @@ inlineToMarkdown opts il@(RawInline f str) = do
       | f `elem` ["markdown", "markdown_github", "markdown_phpextra",
                   "markdown_mmd", "markdown_strict"]
          -> return $ literal str
+    Markua -> do
+        report $ NoTranslation "raw content is writen out in original form and"
```

Comment:
See above

Reply:
Why not use `renderEmpty` here? It emits the right kind of warning.

Reply:
We changed it accordingly